### PR TITLE
added trending page

### PR DESCRIPTION
**Pull request by AbhishekkGautam**
### Trending Page 
- horizontal cards
- sort videos in descending order based on views 

mobile (8)

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
The trending page is looking awesome.
The code seems clean.
Good to go!

File: src/components/HorizontalCard/HorizontalCard.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { MoreOptionsModal } from "../MoreOptionsModal/MoreOptionsModal";
+
+export const HorizontalCard = ({ trendingVideo }) => {
+  const {
+    _id: videoId,
+    title,
+    channelName,
+    views,
+    description,
+  } = trendingVideo;
```
Comment (MiheerTamkhane) [inline src/components/HorizontalCard/HorizontalCard.js:17]: Good to destructure here code looks more readable

File: src/components/VideoListing/VideoListing.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const VideoListing = () => {
                 <div className="info-container">
                   <img className="avatar" src={avatar} alt={channelName} />
                   <div className="video-info">
-                    <h3 className="video-title">{trimExtraChars(title)}</h3>
+                    <h3 className="video-title">{trimExtraChars(title, 50)}</h3>
                     <span>{channelName}</span>
```
Comment (MiheerTamkhane) [inline src/components/VideoListing/VideoListing.js:48]: I liked how you created a trim function. Nice work

**Review by MiheerTamkhane — APPROVED** [review]
The overall code looks nice and clean. also the UI is awsm. 
Good choice or colors , fonts and Icons. NIce work

File: src/components/VideoListing/VideoListing.js (hunk 2 above)
Comment (AbhishekkGautam) [reply src/components/VideoListing/VideoListing.js:48]: Thank you

**Pull request merged**